src/main/java/Duke.java

```diff
@@ -1,10 +1,239 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static int MAX_SIZE = 100;
+    public static Task[] list = new Task[MAX_SIZE];
+    private static int LIST_COUNTER = 0;
```

Comment:
`MAX_SIZE` is a constant so it should be marked with `final`.
`LIST_COUNTER` is not a constant so it should be named using camelCase, i.e. `listCounter`.